layouts/shortcodes/blocks/section.html

```diff
@@ -1,16 +1,14 @@
-{{ $_hugo_config := `{ "version": 1 }` -}}
```

Comment:
by removing this line, this PR also contributes to #939. Can you add this in the issue description, please?

Reply:
Ah yes! Thanks for the reminder, ... and the review! 😄